### PR TITLE
[merged] Compiler warnings

### PR DESCRIPTION
**Pull request by cgwalters**

### CODE REVIEW

**Comment by cgwalters** [comment]
bot, retest this please

File: configure.ac [hunk 1]
```diff
@@ -15,16 +15,19 @@ AC_PROG_CC
 AM_PROG_CC_C_O
 AC_PROG_YACC
 
-changequote(,)dnl
-if test "x$GCC" = "xyes"; then
-  WARN_CFLAGS="-Wall -Wempty-body -Wstrict-prototypes -Werror=missing-prototypes \
```
Comment (giuseppe) [inline configure.ac]: was `-Wempty-body` removed on purpose?

**Comment by cgwalters** [comment]
Readded and made it `-Werror=empty-body` now for consistency.  (We want this whole list to be fatal :bomb: :computer:  errors).

**Comment by giuseppe** [comment]
@rh-atomic-bot r+ 6a12289

**Pull request closed without merging**